src/components/composites/Map/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import {
+  GoogleMap,
+  LoadScript,
+  Marker,
+  InfoWindow,
+} from "@react-google-maps/api";
+import raceTracks from "../../../assets/data/test_data.json";
+import MultiActionAreaCard from "../../muiCard";
+import PlacePhoto from "../../utility/PlacePhoto";
+
+const containerStyle = {
+  width: "100vw",
+  height: "88vh",
```

Comment:
whats the plan with making this responsive for mobile

Reply:
I think it should be responsive in case the client wants to show the demo to a prospective client. I understand that the client wanted a web page and a mobile application, so the prototype of Adobe XD was shown to him as an app.